preprocess_observation_smogon.py

```diff
@@ -98,7 +100,8 @@ def preprocess_observation(I):
                         for i in range(NSC_PLACEHOLDER):
                             retval.append([OFFSET_STATUS_CONDITIONS + relevant_offsets[2] + NUM_STATUS_CONDITIONS * relevant_indices[name] + i, 0])
                     else:
-                        health = int(split_line[-1].split('/')[0])
+                        health = int(split_line[-1].split('/')[0])/([OUR_TEAM_MAXHEALTH, OPPONENT_TEAM_MAXHEALTH]['p2a' in line][relevant_indices[name]])
```

Comment:
Repeated code -- could be refactored.

Reply:
It looks identical. But there are some critical differences. On line 60 we have:

health = int(condition[0])...

On line 89 we have:

health = int(split_line[-2].split('/')[0])...

Which is noticeably different. On line 103 we have:

health = int(split_line[-1].split('/')[0])

Which is similar, but has a different index. This index is because of an if-else statement these are in. In the former case, we believe split_line looks like A, while in the latter we believe split_line looks like B.